hazelcast/src/test/java/com/hazelcast/jet/pipeline/OrderedProcessingMultipleMemberTest.java

```diff
@@ -88,15 +89,12 @@
 
     @BeforeClass
     public static void setupClass() {
-        int testCount = 16;
-        Config config = new Config();
-        for (int idx = 0; idx < testCount; idx++) {
-            EventJournalConfig eventJournalConfig = config
-                    .getMapConfig("test-map-" + idx)
-                    .getEventJournalConfig();
-            eventJournalConfig.setEnabled(true);
-            eventJournalConfig.setCapacity(30000); // 30000/271 ~= 111 item per partition
-        }
+        Config config = smallInstanceConfig();
```

Comment:
Although it extends a bit the scope of PR, I chose to convert this configuration setting  to wildcard here too